diff --git i/info/Makefile.in w/info/Makefile.in
index 4f23ef2..e17c784 100644
--- i/info/Makefile.in
+++ w/info/Makefile.in
@@ -2116,6 +2116,7 @@ BUILT_SOURCES = funs.h
 # Test suite
 CLEANFILES = $(am__append_2)
 info_tests_with_pseudotty = \
+	t/malformed-split.sh \
 	t/adjust-anchors.sh \
 	t/all-only.sh \
 	t/anchor-positions.sh \
@@ -2142,7 +2143,6 @@ info_tests_with_pseudotty = \
 	t/index.sh \
 	t/last-node-not-top-level.sh \
 	t/last-no-history.sh \
-	t/malformed-split.sh \
 	t/menu-sequence.sh \
 	t/multiple-completions.sh \
 	t/next-quoted.sh \
